sesheta/webhooks.py

```diff
@@ -222,23 +222,28 @@ def handle_github_open_pullrequest_merged_successfully(pullrequest: dict) -> Non
     return
 
 
-def _add_size_label(pullrequest: dict) -> None:  # pragma: no cover
-    """Add a size label to a GitHub Pull Request."""
+def _add_required_labels(pullrequest: dict) -> None:  # pragma: no cover
+    """Add the required label to a GitHub Pull Request."""
     if pullrequest["title"].startswith("Automatic update of dependency"):
         return
 
     if pullrequest["state"] == "closed":
         return
 
-    sizeLabel = calculate_pullrequest_size(pullrequest)
+    if pullrequest["title"].startswith("WIP") or pullrequest["title"].startswith("[WIP]"):
+        _LOGGER.debug(f"{pullrequest['title']} is not mergeable, it's work-in-progress!")
+        add_labels(pullrequest["_links"]["issue"]["href"], ["work-in-progress"])
```

Comment:
shall we call the label "do-not-merge/work-in-progess" ? so that we can have other 'do-not-merge' labels like 'needs-rebase' 'needs-reviewer-assigned' ...